cfme/tests/test_rest.py

```diff
@@ -69,6 +73,144 @@ def provision_data(
     return result
 
 
+@pytest.fixture(scope='module')
+def service_catalogs_fixture(request, rest_api):
+    name = fauxfactory.gen_alphanumeric()
+    scl = rest_api.collections.service_catalogs.action.add(
+        name=name,
+        description="description_{}".format(name),
+        service_templates=[]
+    )[0]
+    wait_for(
+        lambda: rest_api.collections.service_catalogs.find_by(name=name),
+        num_sec=180,
+        delay=10,
+    )
+
+    service_catalogs_data = [{
+        "name": "name_{}_{}".format(name, index),
+        "description": "description_{}_{}".format(name, index),
+        "service_templates": []
+    } for index in range(1, 5)]
+
+    scls = rest_api.collections.service_catalogs.action.add(*service_catalogs_data)
+
+    def _finished():
+        scl.reload()
+        scl.action.delete()
+        with error.expected("ActiveRecord::RecordNotFound"):
+            scl.action.delete()
+
+        rest_api.collections.service_catalogs.action.delete(*scls)
+        with error.expected("ActiveRecord::RecordNotFound"):
+            rest_api.collections.service_catalogs.action.delete(*scls)
+
+    request.addfinalizer(_finished)
+    return scls
+
+
+@pytest.mark.usefixtures("logged_in")
+@pytest.fixture(scope='module')
+def dialog_fixture():
+    dialog = "dialog_" + fauxfactory.gen_alphanumeric()
+    element_data = dict(
+        ele_label="ele_" + fauxfactory.gen_alphanumeric(),
+        ele_name=fauxfactory.gen_alphanumeric(),
+        ele_desc="my ele desc",
+        choose_type="Text Box",
+        default_text_box="default value"
+    )
+    service_dialog = ServiceDialog(label=dialog, description="my dialog",
```

Comment:
I personally oppose parenthesis-aligned long parameter list because it often leads to a non-4 indent.

```python
service_dialog = ServiceDialog(
    asdf=ghjk,
    qwertz=uiop)
```